app/src/main/java/com/github/se/eventradar/model/event/Event.kt

```diff
@@ -11,13 +11,19 @@ enum class EventCategory {
   COMMUNITY
 }
 
+data class Ticket(val name: String, val price: Double, val quantity: Int)
```

Comment:
I see that the `quantity` parameter is in the blueprint, but I remember us not coming to a consensus on this because 1) it adds complexity in the QR code scanning process and 2) it was an extra feature. Did we decide as a group to keep this? The blueprint does not mention how the quantity is used.

Reply:
I think this ticket quantity represent the max number of people that can attend the event and not the number of ticket one person can buy. So it should not complexify the scanning or I am forgetting something ?

Reply:
Quantity will be how many tickets are available, 
every time one is purchased it will decrease by 1, at 0 => sold out

we are still keeping: 1 event purchase per user :)